app/src/main/java/org/eyeseetea/malariacare/SplashScreenActivity.java

```diff
@@ -38,9 +40,11 @@ private void init() {
 
         D2.init(this);
         SdkQueries.createDBIndexes();
-
-        //TODO: after mega merge
-        //PostMigration.launchPostMigration();
+        //Added to execute a query in DB, because DBFLow doesn't do any migration until a query
+        // is executed
+        if (PopulateDB.hasMandatoryTables()) {
```

Comment:
@manuelplazaspalacio this provoked that postmigration is not executed because is these moment mandatory tables have not data, it's populate in pull after login.